pkg/logicalbackup/logicalbackup.go

```diff
@@ -38,6 +39,16 @@ const (
 	cDelete
 )
 
+type NameChangeHistory struct {
```

Comment:
maybe something like `NameAtLsn` `NameLsn`
words `Change` and `History` imply to some dynamics, i.e. maps in the struct.
But I don't see any

Reply:
Yep, I will rename it to "Entry".